<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="lwProvince">
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>广东省</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/shengdanwa.png</normaloff>:/images/shengdanwa.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>江苏省</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/shengdanye.png</normaloff>:/images/shengdanye.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>山东省</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/xiaxue.png</normaloff>:/images/xiaxue.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>浙江省</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/xing.png</normaloff>:/images/xing.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbListMode">
       <property name="text">
        <string>图标模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbIconMode">
       <property name="text">
        <string>列表模式</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>当前选中条目：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInsert">
       <property name="text">
        <string>插入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDel">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
